aws/logs_monitoring/lambda_function.py

```diff
@@ -250,7 +255,7 @@ def compileRegex(rule, pattern):
 DD_CUSTOM_TAGS = "ddtags"
 DD_SERVICE = "service"
 DD_HOST = "host"
-DD_FORWARDER_VERSION = "2.3.0"
+DD_FORWARDER_VERSION = "2.3.1"
```

Comment:
sorry I took 2.3.1 in https://github.com/DataDog/datadog-serverless-functions/pull/175

```suggestion
DD_FORWARDER_VERSION = "2.3.2"
```